pkg/handler/handler.go

```diff
@@ -439,6 +522,11 @@ func (h handler) bind(w http.ResponseWriter, r *http.Request, params map[string]
 		return
 	}
 
+	var async bool
+
+	// ignore the error, if async can't be parsed it will be false
+	async, _ = strconv.ParseBool(r.FormValue("accepts_incomplete"))
```

Comment:
Should we turn on ```accepts_incomplete``` if ```LaunchApbOnBind``` is enabled and ```accepts_incomplete``` is not explicitly set to false?  Also, should we have a warning if we have ```LaunchApbOnBind=true``` and ```accepts_incomplete=false``` ?

Reply:
Yeah, I'll add a warning.

Reply:
Actually, `accepts_incomplete` will be given to us from the service-catalog. We can not change its value once we get it since the caller needs to know how to monitor the `last_operation` endpoint.

I will add a warning when  `LaunchApbOnBind=true` and `accepts_incomplete=false` condition occurs. Worst case it means that during synchronous binds we WILL launch the apb, which COULD timeout.